InnerEye/ML/scalar_config.py

```diff
@@ -41,6 +42,7 @@ class EnsembleAggregationType(Enum):
 @unique
 class ScalarLoss(Enum):
     BinaryCrossEntropyWithLogits = "BinaryCrossEntropyWithLogits"
+    # WeightedCrossEntropyWithLogits does not work with 1 class_names, use 2 class_names and 2 outputs to allow weights
```

Comment:
Is this still true? There is logic in BinaryCrossEntropyWithLogits to handle the single class case.

Reply:
This should be removed. At some point I considered not supporting this case